### PR TITLE
change source for flint in travis

### PR DESCRIPTION
**Pull request by srajangarg**

### CODE REVIEW

File: bin/install_travis.sh [hunk 1]
```diff
@@ -81,9 +81,8 @@ if [[ "${WITH_MPFR}" == "yes" ]] || [[ "${WITH_MPC}" == "yes" ]] || [[ "${WITH_A
     fi
 fi
 if [[ "${WITH_ARB}" == "yes" ]] || [[ "${WITH_FLINT}" == "yes" ]]; then
-    wget https://raw.githubusercontent.com/symengine/dependencies/6a42d290071921a0a478c6883fc0ddd709d664c9/flint-2.4.4.tar.gz;
-    tar -xzf flint-2.4.4.tar.gz;
-    cd flint-2.4.4 && ./configure --prefix=$our_install_dir && make -j8 install && cd ..;
+    git clone https://github.com/wbhart/flint2;
+    cd flint2 && ./configure --prefix=$our_install_dir && make -j8 install && cd ..;
```
Comment (isuruf) [inline bin/install_travis.sh]: Can you checkout a specific commit? 44ec3871a555507038fe3838d9c2f6c0ac3064fb

**Comment by srajangarg** [comment]
@isuruf

**Comment by isuruf** [comment]
For the record, this was done to use a bug fix introduced in the git version.

**Comment by srajangarg** [comment]
https://github.com/wbhart/flint2/issues/217 was the bug.

**Pull request merged**